packages/dashboard/src/components/composed/Debugger/Home/EtherscanApiKeyPrompt.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Container,
+  Stack,
+  Flex,
+  Text,
+  Button,
+  Input,
+  createStyles
+} from "@mantine/core";
+import { useInputState, useLocalStorage } from "@mantine/hooks";
+
+const useStyles = createStyles(theme => ({
+  inputGroup: {
+    marginTop: 10,
+    input: {
+      height: 42,
+      borderTopRightRadius: 0,
+      borderBottomRightRadius: 0,
+      border: 0
+    }
+  },
+  reset: {
+    "&:hover": {
+      cursor: "pointer"
+    },
+    "color": "#E03131",
+    "fontSize": 14
+  },
+  halfWidth: {
+    width: "50%"
+  },
+  promptContainer: {
+    width: "50%",
+    borderRadius: 4,
+    backgroundColor:
+      theme.colorScheme === "dark"
+        ? "#082720"
+        : theme.colors["truffle-teal"][0],
+    border: `solid ${theme.colors["truffle-teal"][7]}`,
+    padding: 15
+  },
+  etherscanMessageContainer: {
+    width: "100%"
+  }
+}));
+
+function EtherScanApiKeyPrompt() {
+  const { classes } = useStyles();
+
+  const [etherscanApiKey, setEtherscanApiKey, removeEtherscanApiKey] =
+    useLocalStorage({
+      key: "etherscan-api-key"
+    });
```

Comment:
Reference comment in src/components/composed/Debugger/index.tsx on naming the key.